### PR TITLE
[usage] Include drafts when listing usage summary

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

We need to use the same draft flags between ListUsage and ListUsageSummary, else the metadata about pagination does not match.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: components/usage/pkg/apiv1/usage.go [hunk 1]
```diff
@@ -78,15 +78,17 @@ func (s *UsageService) ListUsage(ctx context.Context, in *v1.ListUsageRequest) (
 	if in.GetPagination().GetPage() > 1 {
 		page = in.GetPagination().GetPage()
 	}
-	var offset int64 = perPage * (page - 1)
+	var offset = perPage * (page - 1)
```
Comment (easyCZ) [inline components/usage/pkg/apiv1/usage.go:81]: drive-by fix, the type is not needed

**Review by svenefftinge — APPROVED** [review]
